quora-api/src/main/java/com/upgrad/quora/api/exception/RestExceptionHandler.java

```diff
@@ -0,0 +1,26 @@
+package com.upgrad.quora.api.exception;
+
+import com.upgrad.quora.api.model.ErrorResponse;
+import com.upgrad.quora.service.exception.SignUpRestrictedException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+
+
+// This annotation allow to write a code which will be made available to all controller class
+@ControllerAdvice
+
```

Comment:
Please remove extra whitespace and add javadoc style comment for this class

Reply:
done